### ISSUE TITLE
[V4] Filters are missing `id`, `createdAt` and `updatedAt`

### ISSUE THREAD

**Issue by ScottAgirs** [strapi/strapi#11516]
### Describe the bug

It seems like the `id` field is missing from the `filters`key in GraphQL.
This prevents the user from requesting for example Posts based the Author id:
```js
query POSTS_Q {
  posts(filters:{user: 1}) {
    data {
      id
      attributes {
        title
      }
    }
  }  
}
```

### Expected behavior

Ability to request Collection Type based on a relational field's content type by its `ID`

### System

- Strapi version: `beta.11`

**Comment by derrickmehaffy** [comment]
@Convly not sure if scott and I are missing something (in which case it probably just needs to be clarified in the docs) but I could not for the life of me figure out how to use `id`, `createdAt`, or `updatedAt`

**Comment by Convly** [comment]
Hey, filtering on `ID` seems to be unavailable indeed, I'll take a look.
However, I've been able to filter using `createdAt` & `updatedAt` (even if given the output, they're likely broken)

**Comment by LucaNerlich** [comment]
Duplicate of #11151

**Comment by derrickmehaffy** [comment]
> Duplicate of #11151

Not entirely a duplicate, filters in the CTM vs in REST/GraphQL aren't quite the same as they call two different APIs (Content API for REST/GraphQL and Admin API for the Admin Panel)

**Comment by LucaNerlich** [comment]
Sorry for that, long day :/

**Comment by Convly** [comment]
Closing as it should be fixed by #11552 (available in the next release). Feel free to reopen the issue if the problem persists.
